spec/rubocop/cop/style/commented_keyword_spec.rb

```diff
@@ -237,6 +237,139 @@ def x(y: "#value")
     RUBY
   end
 
+  it 'does not register an offense for steep annotation of method definition' do
+    expect_no_offenses(<<~RUBY)
+      def x # steep:ignore
+      end # steep:ignore
+
+      def x # steep:ignore MethodBodyTypeMismatch
+      end # steep:ignore MethodBodyTypeMismatch
+
+      class X # steep:ignore
+      end
+
+      class X # steep:ignore UnknownConstant
+      end
+
+      module X # steep:ignore
+      end
+
+      module X # steep:ignore UnknownConstant
+      end
+
+      begin # steep:ignore
+      end
+
+      begin # steep:ignore NoMethod
+      end
+    RUBY
+  end
+
+  it 'registers an offense and corrects for steep annotation of method definition' do
```

Comment:
There are the following concerns regarding the test cases:

1. The spec description is the same as the above `expect_no_offenses`. It seems necessary to add "non" somewhere to differentiate them. (Or something that represents such a difference)
2. "steep" looks like "Steep."  (or `steep:ignore`)
3. Ideally, to maintain consistency with the implementation, these specs for Steep annotation should be written after `RBS::Inline` specs so that the order remains consistent.  

Additionally, I'm concerned that multiple perspectives are mixed within the test cases. Would it be possible to split them under a context like `context 'when Steep annotation is used' do`?

Reply:
I nested this test case in a context block and separated the test cases by keywords like def, class, module, etc.

As you pointed out, the context was mixed.
Thanks!!

Perhaps the test cases for RBS::Inline Annotation could be nested in the same way, but I don't fix it because this PR context is Steep.

Reply:
> Perhaps the test cases for RBS::Inline Annotation could be nested in the same way, but I don't fix it because this PR context is Steep.

Yep, since the purpose is different, and it is not worth modifying existing tests just to alter `git blame`, keeping them as they are would be better choice I think. Thank you!